.github/workflows/e2e.yml

```diff
@@ -32,6 +32,8 @@ jobs:
 
       - name: Setup version of Dart
         uses: dart-lang/setup-dart@d6a63dab3335f427404425de0fbfed4686d93c4f
+        with:
```

Comment:
This reintroduces a dev version (non-stable) of the Dart runtime...

Reply:
Pinned it!